docs/Splashkit/Applications/Meeting Minutes/MM11April.md

```diff
@@ -0,0 +1,70 @@
+# Meeting minutes
+
+Monday: 11/04/2022
+
+Attendees at 4:40: [Lily-yyyy], Nick Agiazis, Anthony George, Sarah Gosling, Riley Nicholas Dellios, [MorgaineBarter], Bella Chhour (at 5:10)
+
+Absent: Huy Pham
+
+Next meeting: 14/4/2022 6pm Thursday
+
+Minutes by : [MorgaineBarter] and [AnthonyGeorge]

```

Comment:
Curious as to what the brackets mean?

Reply:
I think that it was from the template given. They had brackets for next to the names